package.json

```diff
@@ -12,7 +12,7 @@
     "react-firestore": "^1.0.1",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.2.0"
+    "react-scripts": "^3.3.1"
```

Comment:
A good rule of thumb is to explain why you are updating a dependency if it changes in your PR. For well-established projects, even small dependency changes can cause issues. So, people will discuss the good with the bad on needing to update.

Reply:
Thanks for pointing this one out and highlighting the need to explain in the future. I think this update was a result of an `npm audit fix`, but I'm not 100% sure. I wouldn't anticipate an issue following this change, but will monitor any similar updates more closely next time.